src/asciireader.cpp

```diff
@@ -177,17 +178,18 @@ SamplePack* AsciiReader::parseLine(const QString& line) const
     auto samples = new SamplePack(1, numComingChannels);
     for (unsigned ci = 0; ci < numComingChannels; ci++)
     {
+	strippedValue = separatedValues[ci].split(':', QString::SkipEmptyParts).back();
```

Comment:
Also instead of `split` it is cleaner (and I'm guessing faster) to use [QString::section](https://doc.qt.io/qt-5/qstring.html#section). Like this:

```
value.section(':', 1); 
```